pom.xml

```diff
@@ -220,11 +219,6 @@
             </dependency>
             <!-- Include jaxb runtime dependencies which have been removed
```

Comment:
Do we still need this comment when the corresponding dependency is removed?

Reply:
I don't know if this comment is only for the direct following mentioned dependency or more. I can remove it if you want.